horizon: install lbaas horizon dashboard (SOC-10883)

stefannica opened the pull request:
Add the LBaaS dashboard to the UI if lbaas is enabled in the
neutron barclamp.

jsuchome commented on chef/cookbooks/horizon/recipes/server.rb:
```diff
@@ -103,6 +103,26 @@
+  case node[:platform_family]
+  when "suse"
+    "openstack-horizon-plugin-neutron-lbaas-ui"
+  when "rhel"
```
Does the separate branch for RHEL package make any sense? I know we have it elsewhere, but ...

jsuchome replied:
I don't think so. And AFAIK it never ran there. We were just trying to be good open source citizens. Now it matters even less, that's why I wouldn't even add such new code.... but it does not hurt, so no need to rebase it just because of this

stefannica replied:
I removed the rhel section :-)

stefannica commented:
I fixed the hound complaints and I also removed any Octavia references from this PR, because this should be just about adding the lbaas dashboard to horizon. I will address Octavia related logic in another PR, where I will also address adding the Octavia dashboard.

stefannica commented:
Adding the Octavia horizon dashboard with all its related concerns is handled in #2311
